subiquity/common/apidef.py

```diff
@@ -168,6 +170,14 @@ class info:
             def GET(dev_name: str) -> str: ...
 
     class storage:
+        class guided:
+            def GET(wait: bool = False) -> GuidedStorageResponse:
```

Comment:
Just checking I understand this correctly. I call GET on 'storage/guided' and get a list of disks back in GuidedStorageResponse. I then POST a GuidedChoice containing the disk I want to install to. Simple as that?

Reply:
Yes, exactly. If pass don't pass wait or pass wait=false as a url param, you might get a result with probe_status == PENDING, but if you pass wait=true the method will block until the list of disks is known. In desktoppy cases I really wouldn't expect the probe to take more than a trivial amount of time though. (It can be slow with virtual ISOs and stuff like that).

Reply:
Awesome, thanks for clarifying.